src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -997,11 +1120,24 @@ public string P
 ");
             var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
             comp.TestOnlyCompilationData = accessorBindingData;
+            if (member == "private int field;")
+            {
+                comp.VerifyDiagnostics(
+                    // (6,17): warning CS0649: Field 'C.field' is never assigned to, and will always have its default value 0
```

Comment:
>warning CS0649 [](http://example.com/codeflow?start=31&length=14)

We can simply suppress this warning in the test scenario with a ```#pragma``` directive.

Reply:
Yeah that's much clearer. Thanks!